lib/matplotlib/backends/__init__.py

```diff
@@ -14,22 +14,21 @@
            'new_figure_manager', 'backend_version']
 
 backend = matplotlib.get_backend() # validates, to match all_backends
+if backend.startswith('module://'):
+    backend_name = backend[9:]
+else:
+    backend_name = 'matplotlib.backends.backend_' + backend.lower()
 
-def pylab_setup():
-    'return new_figure_manager, draw_if_interactive and show for pylab'
+def get_backend():
```

Comment:
OK. That might be a problem.
But still worth to think about
On 27 Jun 2015 7:19 pm, "Thomas A Caswell" <notifications@github.com> wrote:

> In lib/matplotlib/backends/__init__.py
> <https://github.com/matplotlib/matplotlib/pull/4143#discussion_r33417385>:
>
> >
> > -def pylab_setup():
> > -    'return new_figure_manager, draw_if_interactive and show for pylab'
> > +def get_backend():
>
> For one thing, doing that requires doing some non-standard things to
> sys.module (
> http://stackoverflow.com/questions/880530/can-python-modules-have-properties-the-same-way-that-objects-can
> ).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4143/files#r33417385>.
>